docs/products/mysql/howto/connect-from-mysqlx.rst

```diff
@@ -0,0 +1,59 @@
+Connect to MySQL using MySQLx with Python
+=========================================
+
+Enabling MySQLx protocol support allows customers to use their MySQL instance as a document store. This example shows how to connect to your Aiven for MySQL instance using MySQLx protocol.
+
+.. note::
+
+    MySQL initially provided support for X-DevAPI (MySQLx) in v5.7.12 as an optional extension that can be installed by the user. On the MySQL v8.0+, the X-DevAPI is supported by default.
+
+Variables
+'''''''''
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Variable
+     - Description
+   * - ``SERVICE_URI``
+     - Connection information about your Aiven for MySQLx service.
+   * - ``MYSQLX_PASSWORD``
+     - Password for ``avnadmin`` user
+
+Prerequisites
+'''''''''''''
+
+* Python 3.7 or later
+  
+* A ``mysqlx`` python library installed::
+     
+     pip install mysql-connector-python
+
+* An Aiven account with a service running. Click on **MySQLx** tab under **Connection information**, and copy ``SERVICE_URI`` and ``Port`` information.
```

Comment:
I think it make sense to add PORT as additional variable as well. and specify that it is port for mysqlx, since it is different from default one

Reply:
I'm not sure because when you test, you will see that the `SERVICE_URI` already includes the `PORT`. I think is very confusing to rename things different of the UI and do the split of the string ourselves etc.